docker/testnet/testnet-host.env

```diff
@@ -0,0 +1,7 @@
+# Default Port Selection [These are only host side mappings from docker environment]
+TM_REST_API_PORT=1317
+TM_GRPC_SERVER_PORT=9090
+TM_GRPC_API_GATEWAY_PORT=9191
+TM_P2P_PORT=26656
+TM_JSON_RPC_PORT=26657
+CHEQD_TESTNET_VERSION=v0.2.7
```

Comment:
Will now be bumped to v0.3.1

Reply:
Should I change that beforehand? Proceeding.